ReqIFSharp/AttributeValue/AttributeValueDate.cs

```diff
@@ -132,7 +132,14 @@ protected override void SetAttributeDefinition(AttributeDefinition attributeDefi
         public override void ReadXml(XmlReader reader)
         {
             var value = reader["THE-VALUE"];
-            this.TheValue = XmlConvert.ToDateTime(value, XmlDateTimeSerializationMode.Utc);
+            try
```

Comment:
I am not in favour of handling it this way. 

In the event the reqif document is read with validation (using the xds's) then the schema validation will complain. If the document is not read using validation, this should throw an exception, the document is then incorrect and in my opinion we should stop parsing and let XmlConvert throw. 

please revert.